API.md

```diff
@@ -65,13 +65,13 @@ After running the previous command, you should see output similar to the followi
 ```
 *   Trying [::1]:9000...
 * Connected to localhost (::1) port 9000
-> GET /irods-http-api/0.1.0/authenticate HTTP/1.1
+> GET /irods-http-api/0.2.0/authenticate HTTP/1.1
```

Comment:
Same thing here. The version number could be replaced with `<version>` since this section is for demonstration purposes.

Reply:
agreed.